apps/parsley/src/components/Search/SearchBar/index.tsx

```diff
@@ -226,6 +226,10 @@ const StyledInput = styled(TextInputWithGlyph)`
     border-left: 0;
     padding-left: 42px;
   }
```

Comment:
Could we also fix the border rounding on this component since it was also broken when this broke? 
This component should look like this
<img width="889" alt="image" src="https://github.com/evergreen-ci/ui/assets/4605522/1b2fd3f4-2ced-473e-8127-b2f3ccf7bf55">